internal/bin/cldev

```diff
@@ -7,10 +7,11 @@
 # 3. cd solidity && truffle migrate && cd ..
 # 4. ./internal/bin/cldev
 
+cd "$(dirname "$0")"
```

Comment:
Surely this has no effect since the `pushd` on line 23 would just overwrite it.

Reply:
Doesn't the `popd` on line 28 place it back?